### PR TITLE
improve about page

### PR DESCRIPTION
**Pull request by ryanatkn**
We haven't touched the "about" page text in a very long time, and some of it looks rather off, so I'm trying to do some limited cleanup.

### CODE REVIEW

File: src/routes/about.svelte [hunk 1]
```diff
@@ -15,37 +15,67 @@
...
-		<li>blogs and web publishing</li>
-		<li>commerce</li>
-		<li>maybe one day, galaxies of interesting communities to explore</li>
-		<li>client software that users fully control</li>
-		<li>client software that works with many service providers in a single app</li>
-		<li>server software that's easy to self-host</li>
-		<li>and more! much more</li>
+		<li>
+			<details>
+				<summary>consumption</summary> much of the time you'll consume content, like of familiar
```
Comment (greatbacon) [inline src/routes/about.svelte:28]: I would probably put `consumption` under `create` (even if the majority of the user base will be engaging in this manner)

Comment (greatbacon) [reply src/routes/about.svelte:28]: I'd also probably phrase it as `the majority of the time on felt, users will be engaging with familiar content like images, videos...` etc.

File: src/routes/about.svelte [hunk 2]
```diff
@@ -15,37 +15,67 @@
...
+				media types -- images, videos, chats, forums, podcasts, lists, polls -- and a huge
+				possibility space to explore for things like novel discussion types like deliberations,
```
Comment (greatbacon) [inline src/routes/about.svelte:30]: `as well as a huge space of possibility to explore more novel forms like...`

File: src/routes/about.svelte (hunk 1 above)
Comment (ryanatkn) [reply src/routes/about.svelte:28]: Agreed, also I cut this section for now, thinking we should draft something together.

**Review by greatbacon — APPROVED** [review]

**Comment by ryanatkn** [comment]
Sorry this was rougher than I thought when I opened the PR -- I made the changes you mentioned and some others. I think I'm done making major changes to it. Kinda wondering if we should delete any sections, like the collaboration one. I changed it to be expanded by default.

File: src/routes/about.svelte [hunk 3]
```diff
@@ -15,37 +15,63 @@
...
+			sandbox
+		</li>
+		<li>
+			<strong>consumption</strong>: the majority of time on Felt, users will be engaging with
+			familiar content like text, images, videos, chats, forums, podcasts, lists, and polls, as well
+			as underexplored novel and novel forms like structured deliberations, virtual spaces with
+			audio/visual streams,
+			<a href="https://en.wikipedia.org/wiki/Twitch_Plays_Pok%C3%A9mon">TPP</a>-inspired social
+			experiences, and, getting back to the real world, events — imagine going to lunch with
+			friends!
```
Comment (ryanatkn) [inline src/routes/about.svelte:39]: I tried to make this section point back to IRL, but I feel it's missing some important nod to the idea that Felt is not designed to maximize engagement, and instead should use design to reduce it in many cases.

**Pull request merged**